src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -121,10 +121,48 @@ namespace ngraph
                 void CPUAssignment::ASSIGN_DECL(ngraph::op::ConvolutionRelu)
                 {
                     auto convolution = static_cast<op::ConvolutionRelu*>(node);
-                    auto op_annotations =
-                        std::make_shared<ngraph::runtime::cpu::CPUOpAnnotations>();
-                    op_annotations->set_mkldnn_op(true);
-                    convolution->set_op_annotations(op_annotations);
+
+                    auto arg0_rank = node->get_input_shape(0).size();
```

Comment:
the checks seem to be same for `ConvolutionRelu` and `ConvolutionBiasRelu`. Could we refactor the common logic into a helper?